tools/esp-api/espshell.cpp

```diff
@@ -21,6 +21,7 @@
 #include <cstdio>
 #include <cstring>
 #include <iostream>
+#include <sstream>
```

Comment:
Unnecessary since you reverted to std::string. Not harmful,; doesn't need to be fixed.